typings/index.d.ts

```diff
@@ -0,0 +1,144 @@
+declare module "discord-giveaways" {
+    import { EventEmitter } from "events";
+    import { Client, PermissionResolvable, ColorResolvable, User, Snowflake, GuildMember, TextChannel, MessageReaction, Message } from "discord.js";
+
+    export const version: string;
+    export class GiveawaysManager extends EventEmitter {
+        constructor(client: Client, options?: GiveawaysManagerOptions)
+
+        public client: Client;
+        public giveaways: Giveaway[];
+        public options: GiveawaysManagerOptions;
+        public ready: boolean;
+        public v12: boolean;
+
+        public delete(messageID: Snowflake, doNotDeleteMessage?: boolean): Promise<void>
+        // @ts-ignore-next-line
+        public async deleteGiveaway(messageID: Snowflake): Promise<void>
+        public edit(messageID: Snowflake, options: GiveawayEditOptions): Promise<Giveaway>;
+        public end(messageID: Snowflake): Promise<GuildMember[]>
+        public reroll(messageID: Snowflake, options?: GiveawayRerollOptions): Promise<GuildMember[]>
+        public start(channel: TextChannel, options: GiveawayStartOptions): Promise<Giveaway>;
+
+        public on<K extends keyof GiveawaysManagerEvents>(event: K, listener: (...args: GiveawaysManagerEvents[K]) => void): this;
+
+        public once<K extends keyof GiveawaysManagerEvents>(event: K, listener: (...args: GiveawaysManagerEvents[K]) => void): this;
+
+        public emit<K extends keyof GiveawaysManagerEvents>(event: K, ...args: GiveawaysManagerEvents[K]): boolean;
+    }
+    interface GiveawaysManagerOptions {
+        storage?: string;
+        updateCountdownEvery?: number ;
+        DJSlib?: "v12" | "v11";
+        default?: GiveawayStartOptions;
+        botsCanWin?: boolean;
+        exemptPermissions?: PermissionResolvable[];
+        exemptMembers?: () => boolean;
```

Comment:
I don't know this is valid in TypeScript but I think it is
```suggestion
        exemptMembers?: (member: GuildMember) => boolean;
```